sql/migrations/030-create-rdr-patient-status-log-table.sql

```diff
@@ -1,6 +0,0 @@
-CREATE TABLE `rdr_patient_status_log` (
```

Comment:
Could probably do this as `DROP TABLE IF EXISTS ...`, but since there are only a handful of developers and this isn't in use, it's likely fine to simply remove the migration.